uwsgi.ini

```diff
@@ -33,3 +33,9 @@ endif =
 if-not-env = DEV_ENV
 memory-report = true
 endif =
+if-env = UWSGI_SOCKET_TIMEOUT
+socket-timeout = %(_)
+endif =
+if-not-env = UWSGI_SOCKET_TIMEOUT
+socket-timeout=30
```

Comment:
My only nit here is the inconsistent spacing between this and the `if-env`. Otherwise :+1: